plugins/rapid7_insightidr/Dockerfile

```diff
@@ -1,24 +1,18 @@
-FROM rapid7/insightconnect-python-3-38-slim-plugin:4
-# Refer to the following documentation for available SDK parent images: https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide
+FROM rapid7/insightconnect-python-3-38-plugin:4
```

Comment:
update to 5, please

Reply:
this has been updated to use the newer sdk version